src/app.js

```diff
@@ -1,16 +1,43 @@
-import { DirWatcher, Importer, logger, Streams } from './utils';
+import express from 'express';
+import { ProductController, ReviewController, UserController } from './controllers';
+import { ExpressMiddleware, echoServer, htmlServer, jsonServer, plainTextServer, logger } from './middlewares';
+import { productRouter, userRouter } from './routes';
 
-const dirWatcher = new DirWatcher();
+const app = express();
 
-dirWatcher.watch('./data/**/*.csv', 10000);
-dirWatcher.on('dirwatcher:changed', (path) => {
-  (async () => {
-    const result = await Importer.import(path);
-    logger.info(`ASYNC ${path} ${result}`);
-  })();
+plainTextServer(8091);
```

Comment:
I'd move all these servers initialization to index.js. Because they are not connected with rest server.